pkg/scheduler/algorithm/priorities/test_util.go

```diff
@@ -59,3 +59,19 @@ func priorityFunction(mapFn algorithm.PriorityMapFunction, reduceFn algorithm.Pr
 		return result, nil
 	}
 }
+
+func makeNodeScarceResource(node string, milliCPU, memory int64, scarceResource map[string]int64) *v1.Node {
```

Comment:
this part keeps confusing me. In my mind, a scarce resource is any resource which requests bin packing, whether extended or core

Reply:
yes updated it from scarceResource => extendedResource